src/lib/storage-utils.js

```diff
@@ -71,11 +71,34 @@ function cleanUpReadme(version: Version): Version {
   return version;
 }
 
+function getLatestReadme(pkg: Package): string {
```

Comment:
`getLatestReadme` function requires a unit test in a new file. I suggest `test/unit/latest.readme.spec.js`